metricflow/model/parsing/schemas.py

```diff
@@ -1,5 +1,7 @@
 from jsonschema import RefResolver, Draft7Validator
 
+from metricflow.model.parsing.yaml_loader import PARSING_CONTEXT_KEY
```

Comment:
Per outside discussion, I think we can probably set an internal standard that keys surrounded by (or at least starting with) double-underscore are for internal use only. It's consistent with a lot of other usages as well.

It might be good to document that. Maybe we can put it over wherever we have PARSING_CONTEXT_KEY. I'm not sure if it should be a random module-level constant, but I guess there it is. Moving it seems like a lot of bother for not much benefit at this point.